src/mono/mono/mini/interp/mintops.def

```diff
@@ -552,6 +552,11 @@ OPDEF(MINT_NEG_R8, "neg.r8", 3, 1, 1, MintOpNoArgs)
 OPDEF(MINT_NOT_I4, "not.i4", 3, 1, 1, MintOpNoArgs)
 OPDEF(MINT_NOT_I8, "not.i8", 3, 1, 1, MintOpNoArgs)
 
+OPDEF(MINT_BITCAST_I4_R4, "bitcast.i4.r4", 3, 1, 1, MintOpNoArgs)
+OPDEF(MINT_BITCAST_I8_R8, "bitcast.i8.r8", 3, 1, 1, MintOpNoArgs)
+OPDEF(MINT_BITCAST_R4_I4, "bitcast.r4.i4", 3, 1, 1, MintOpNoArgs)
+OPDEF(MINT_BITCAST_R8_I8, "bitcast.r8.i8", 3, 1, 1, MintOpNoArgs)
```

Comment:
Don't understand why the type matters. Sounds like we should just emit `MINT_MOV_4` or `MINT_MOV_8`.

Reply:
I really did just extract the changes from the linked pr and try to get them working as a first step.  I think the intent was to make sure the jiterp sees the reinterpret but I'm not sure it matters in a practical sense?

cc @kg